apps/metadata-editor-e2e/src/e2e/delete.cy.ts

```diff
@@ -0,0 +1,62 @@
+describe('delete', () => {
+  beforeEach(() => {
+    cy.login('admin', 'admin', false)
+    cy.visit('/catalog/search')
+  })
+
+  describe('record with draft', () => {
+    it('should delete the record, delete its associated draft and refresh the interface', () => {
+      // First create a record and its draft
+      cy.get('[data-cy="create-record"]').click()
+      cy.get('gn-ui-form-field[ng-reflect-model=abstract] textarea').type(
+        'record abstract'
+      )
+      cy.get('md-editor-publish-button').click()
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(1000) // waiting for publication
```

Comment:
Ok I understand better why you added the wait here; it makes sense, you might be able to replace it with a named request like in https://docs.cypress.io/api/commands/wait#Wait-for-a-specific-request-to-respond but it's not such a big deal IMO